README.md

```diff
@@ -236,6 +236,23 @@ class Item(OrderedModel):
     objects = ItemManager()
 ```
 
+
+Custom ordering field
+---------------------
+Extending `OrderedModel` creates a `models.PositiveIntegerField` field called `order` and the appropriate migrations. If you wish to use an existing model field to store the ordering, you can set the attribute `order_field_name` to match your field name:
+
+```python
+class MyModel(OrderedModelBase):
+    ...
```

Comment:
Mb add the `sort_order` field here to make it more explicit?

Reply:
Agreed - fixed

Reply:
Agreed, pushed a fix.